app/views/exam_templates/_boot.js.erb

```diff
@@ -1,3 +1,32 @@
 $(document).ready(function() {
   window.modal_create_new   = new ModalMarkus('#create_new_template');
 });
+
+function add_template_division(id) {
+    var new_id = new Date().getTime();
+    var input_id = 'exam_template_template_divisions_attributes' + new_id;
+    var new_division_row = `
+        <li id="${input_id}_holder" class="new">
+          <span class="label">
+            <input type="text" required="required" name="exam_template[template_divisions_attributes][${new_id}][label]">
+          </span>
+          <span class="start">
+            <input type="number" required="required" name="exam_template[template_divisions_attributes][${new_id}][start]">
+          </span>
+          <span class="end">
+            <input type="number" required="required" name="exam_template[template_divisions_attributes][${new_id}][end]">
+          </span>
+          <span class="delete">
+            <a onclick="remove_template_division(\'${input_id}\'); return false;">
```

Comment:
Eh, I think you can inline the function here. Something like `$(this).closest(li).remove()` should work.